examples/coding-template/Depricated-MultipleImplementationExample-Template.html

```diff
@@ -20,7 +20,7 @@
   The name should be unique if the pattern will have multiple implementations,
   e.g., "Simple Two-State Checkbox" or "Three-State Checkbox".
   -->
-<title>EXAMPLE_NAME | WAI-ARIA Authoring Practices 1.1</title>
```

Comment:
@mcking65 is there a reason why this is deprecated? And should we remove it?

Reply:
The group decided that we should prefer only one example per page. We could consider removing it now. A separate issue is breaking apart the pages that currently have multiple.

Reply:
Excellent, have made a new issue for that: https://github.com/w3c/aria-practices/issues/1235